ts/kpt-functions/src/io.ts

```diff
@@ -36,6 +36,8 @@ const YAML_STYLE: DumpOptions = {
   // Either reason about not having 'undefined' in all cases OR
   // only skip 'undefined'.
   skipInvalid: true,
+  // increase lineWidth from default of 80 to avoid reformatting
+  lineWidth: 120,
```

Comment:
is this what kpt CLI uses? Any follow up for other styling consistency between the two?

Reply:
@phanimarupaka checked kpt and found that it doesn't have any line width limit so we agreed on 120.

I found another inconsistency where kpt keeps single quotes while sdk removes them but there is no clear way to tell DumpOptions to do this.